spendsterIOS/UserDefaults/UserDefaultsStorage.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  UserDefaultsStorage.swift
+//  spendsterIOS
+//
+//  Created by Dmytro Holovko on 5/14/19.
+//  Copyright © 2019 KeyToTech. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+class UserDefaultsStorage: UserStorage {
+    private let defaults = UserDefaults.standard
+    private let userKey = "userkey"
+    private let imageKey = "profileImage"
+    
+    func readImageData() throws -> Data {
```

Comment:
@Thongvor you can use image url as String and load it via https://github.com/Alamofire/AlamofireImage or some other lib. For now, let's keep it as it is. But in the future, please use AlamofireImage to load photos.

Reply:
Ok